indico/modules/rb/client/js/modules/calendar/CalendarListView.jsx

```diff
@@ -101,26 +110,58 @@ class CalendarListView extends React.Component {
           {moment(day, 'YYYY-MM-DD').format('dddd, LL')}
         </Header>
         <Card.Group itemsPerRow={4} stackable styleName="day-cards-container">
-          {bookings.map(this.renderBooking)}
+          {bookings.map(booking => this.renderBooking(booking, day, linkData))}
         </Card.Group>
         <Divider hidden />
       </div>
     );
   };
 
-  renderBooking = booking => {
+  renderLink = (booking, day, linkData, linkBookingOccurrence) => {
+    if (linkData) {
+      const {reservation} = booking;
+      const boundaries = [moment(linkData.startDt), moment(linkData.endDt)];
+      const datesMatch =
+        moment(booking.startDt).isBetween(...boundaries, undefined, '[]') &&
+        moment(booking.endDt).isBetween(...boundaries, undefined, '[]');
+      const linkBtn = (
+        <Button
+          icon={<Icon name="linkify" />}
+          primary
+          disabled={!!booking.link || !datesMatch}
+          size="small"
+          circular
+          onClick={() =>
+            linkBookingOccurrence(reservation.id, day, linkData.id, () =>
+              this.refetchActiveBookings(false)
+            )
+          }
+        />
+      );
+      return (
+        <div style={{position: 'absolute', top: '5px', right: '5px'}}>
```

Comment:
With longer room names, there's an overlap between them and the link button:
![image](https://github.com/indico/indico/assets/8739637/7c6e42a8-d7d2-44a2-819e-0ce4c5154606)

Reply:
I moved it a bit down so that it does not overlap.